content/en/service_management/service_level_objectives/guide/slo_types_comparison.md

```diff
@@ -0,0 +1,45 @@
+---
+title: SLO Type Comparison
+kind: Guide
+further_reading:
+- link: "/service_management/service_level_objectives/"
+  tag: "Documentation"
+  text: "Overview of Service Level Objectives"
+- link: "/service_management/service_level_objectives/metric/"
+  tag: "Documentation"
+  text: "Metric-based SLOs"
+- link: "/service_management/service_level_objectives/monitor/"
+  tag: "Documentation"
+  text: "Monitor-based SLOs"
+---
+
+## Overview
+
+Datadog Service Level Objectives (SLOs) are measured in the following ways: By Count (Metric-based), By Monitor Uptime (Monitor-based), and by Time Slices. Compare what functionalities are supported by each type to see which SLO meets your use case.
+
```

Comment:
I think it would be helpful to have some sort of high level intro before the full chart. I had an intro in https://docs.google.com/document/d/1t6vHaUUHilTgpKhoaA1aUgSn9DMsuCVCEIEiC9A1OJM/edit - open to adjusting it but just want a quick description of each SLO type

Reply:
Ah I see you added it to the SLO overview page, should we add it here too?